Find404.scala

```diff
@@ -0,0 +1,61 @@
+import java.io._
+import scala.io.Source._
+import scala.io.StdIn._
+import scala.collection.mutable.ArrayBuffer
+
+var fileSet = ArrayBuffer[File]()
+var ignoreSet = ArrayBuffer[String]()
+val directorySet = new File(args(0))
+if(directorySet.exists() == false) {
+    println("This directory does not exist")
+} else {
+    fileSet = exploreDirect(directorySet)
+    ignoreSet = readFile(searchFor(fileSet, ".gitignore"))
+}
+
+
+def exploreDirect(directory: File): ArrayBuffer[File] = {
+    var subFileSet = ArrayBuffer[File]()
+    directory.listFiles().foreach {
+        x: File => 
+        if(x.isFile()) subFileSet += x
+        else subFileSet ++= exploreDirect(x)
+    }
+    subFileSet
+}
+
+def searchFor(fileSet: ArrayBuffer[File], name: String): File = {
```

Comment:
Remove this for now, laser focus on the first 3 bullet points

Reply:
All this code will be in git's history whenever we're ready to work on it. But for now remove and recommit and repush.